### ISSUE TITLE
Errors installing on Ubuntu 18.04

### ISSUE THREAD

**Issue by userj** [bhdouglass/clickable#64]
Following the instructions from github readme.
`sudo apt install clickable`  caused a crash, and something didn't get installed in `/usr/bin/clickable`

I reinstalled clickable without the error, but when running
`clickable setup-docker`

I get this:
```

Traceback (most recent call last):
  File "/usr/bin/clickable", line 11, in <module>
    load_entry_point('clickable==4.2.2', 'console_scripts', 'clickable')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 480, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2693, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2324, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2330, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3/dist-packages/clickable.py", line 16, in <module>
    from distutils.dir_util import copy_tree
ModuleNotFoundError: No module named 'distutils.dir_util'
```

**Comment by bhdouglass** [comment]
@JBBgameich are we missing a dependency here?

**Comment by JBBgameich** [comment]
Sounds like that. @userj could you try installing python3-distutils?

**Comment by userj** [comment]
@JBBgameich Yes, that solved it. I started with Ubuntu 18.04 minimal installation, so it might not include that by default.
Thanks!

**Comment by JBBgameich** [comment]
Okay, thanks for testing! I'm glad it works now.  I'll add the dependency to the package so no manual installing is neccesary anymore.

**Comment by JBBgameich** [comment]
Fixed in 1317a290b37dd95e473ed8317e9edfc6d6a33476.

**Comment by bhdouglass** [comment]
Thanks guys this fix will go out in the next release!
